plugins/modules/aws_ssm_parameter_store.py

```diff
@@ -65,10 +65,14 @@
     choices: ['never', 'changed', 'always']
     default: changed
     type: str
-author:
```

Comment:
Why removing the other authors?

Reply:
02:06 ERROR: Found 1 validate-modules issue(s) which need to be resolved:
02:06 ERROR: plugins/modules/aws_ssm_parameter_store.py:0:0: invalid-documentation: DOCUMENTATION.author: Invalid author for dictionary value @ data['author']. Got ['Davinder Pal ( @116davinder )', 'Nathan Webster (@nathanwebsterdotme)', 'Bill Wang (@ozbillwang) <ozbillwang@gmail.com>', 'Michael De La Rue (@mikedlr)'] (75%)

ref: https://app.shippable.com/github/ansible-collections/community.aws/runs/1040/1/console

Reply:
Just grep in plugins/modules `grep -rn -A 4  author *`.  
Maybe the missing indent of the elements is wrong?

Reply:
I will give one more try for it.